modules/installation-and-upgrade/pages/client-requirements.adoc

```diff
@@ -75,6 +75,10 @@ ifeval::[{suma-content} == true]
 | {x86_64}, {aarch64}
 | {check}
 
+| {raspberrypios} 12
```

Comment:
This is in the wrong table. If you open the whole `.adoc` you will see this is the table for SUSE Manager, but there's another for Uyuni below.

I just noticed by having a look at the render.